### PR TITLE
Misc mapping updates

### PR DESCRIPTION
**Pull request by helencarnate**
Relatively minor updates to basic lighting, basic audio, and MMA2 user guides. All successfully tested in local build.

### CODE REVIEW

File: wiki/mapping/basic-audio.md [hunk 1]
```diff
@@ -63,10 +93,10 @@ This is to confirm that the BPM you have found online or manually matches the au
 3. Place the cursor at the start of the new track (Click on the track and press your Home key) and then click `Generate menu > Rhythm Track…`
 4. Input the nearest whole BPM you got earlier into the `Tempo (bpm)` field and enter the duration of the song in the optional `Rhythm track duration` field (the duration is displayed at the top right of the timeline).
 5. Copy the other recommended settings below:
-![Audacity Rhythm Track Menu](https://i.imgur.com/jC6OU0i.png)
+![Audacity Rhythm Track Menu](./images/clicktrack.png)
```
Comment (megalon) [inline wiki/mapping/basic-audio.md]: This file did not load for me. Does it load for you?
image

Comment (megalon) [reply wiki/mapping/basic-audio.md]: image

Comment (megalon) [reply wiki/mapping/basic-audio.md]: I've noticed that `click-track-1.png` and `clicktrack.png` are the same file. This loads with `click-track-1.png`.
I wonder if VuePress doesn't like that there are two nearly identical files with different names.

**Comment by megalon** [comment]
Looks good aside from that one image I pointed out.
I'm not sure why that's happening. Let me know if my suggestion works for you as well

File: wiki/mapping/basic-audio.md (hunk 1 above)
Comment (helencarnate) [reply wiki/mapping/basic-audio.md]: Both versions of the image are building locally. I have swapped to click-track-1.png (to be consistent with later images) and deleted the problematic version... pushing a commit momentarily.
image

**Pull request merged**